unity-renderer/Assets/DCLServices/CameraReelService/CameraReelNetworkService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using System.Threading;
+using UI.InWorldCamera.Scripts;
+
+namespace DCLServices.CameraReelService
+{
+    public interface ICameraReelNetworkService : IService
```

Comment:
probably it's more a personal thing, but I'd rather have the interfaces in a separate file, this also allows us to move interfaces in a separate assembly definition if we start having issues with them and get in the cyclic references situations

Reply:
I also like them separately :) Just followed example of Quests service